Isabelle/Utilities/ROOT

```diff
@@ -0,0 +1,11 @@
+chapter Ouroboros
+
+session Utilities (ouroboros) = HOL +
+  description
+    \<open>General-purpose utilities.\<close>
+  sessions
+    "HOL-Eisbach"
+  theories
+    Utilities
```

Comment:
Given that the new session is named `Utilities`, I think that the theory should be renamed to something different than `Utilities` (maybe `Proof_Methods` or whatever you intend the file to be a container for). That way we are stressing the fact that the `Utilities` session may contain several utility theories.

Reply:
The only utilities planned to be added so far are proof methods for reasoning with equivalences. My intention was to put these into the same theory as the existing proof method `under`, since all proof methods together will not be so many. I think, when doing so it would be most sensible to name the theory like the session.

That said, it might be better to put the `under` proof method and the proof methods for equivalence reasoning into different theories, as they are not that closely related.